pkg/assets/scheme.go

```diff
@@ -27,4 +28,7 @@ func init() {
 	if err := sccv1.AddToScheme(scheme); err != nil {
 		panic(err)
 	}
+	if err := arv1.AddToScheme(scheme); err != nil {
```

Comment:
Why is this necessary? Should we add a comment to explain?

Reply:
This is necessary to make deletion of `arv1.ValidatingWebhookConfiguration` in `deleteCSIWebhook` working. Standard kubernetes+Go procedure